src/pages/main/components/product-grid/product-grid.tsx

```diff
@@ -61,7 +61,46 @@ const ProductGrid = (props: IProps) => {
           </Box>
         </>
       ) : (
-        <>Loading data ...</>
+        <>
+          <Box>
+            <Grid container justify="space-between">
+              <Grid item sm={6} xs={12}>
+                <Typography variant="h5">Најнови производи</Typography>
+              </Grid>
+              <Hidden xsDown>
+                <Grid container item xs={6} justify="flex-end" alignItems="flex-end">
+                  <Box component="span">
+                    <Typography variant="body2">прикажи ги сите {' >'}</Typography>
+                  </Box>
+                </Grid>
+              </Hidden>
+              <Hidden smUp>
+                <Grid container item xs={12} alignItems="flex-end">
+                  <Box component="span" mt={1}>
+                    <Typography variant="body2">прикажи ги сите {' >'}</Typography>
+                  </Box>
+                </Grid>
+              </Hidden>
+            </Grid>
+          </Box>
+          <Box mt={3} pb={3}>
+            <Grid container direction="row" justify="center" alignItems="stretch" spacing={2}>
+              {cards.map((res, index) =>
```

Comment:
you have `.slice(0, 8)` function for getting only 8 elements from the array

Reply:
Yes, I think we made this way earlier, will fix it